kzz7yh-f49503/cod-ve09v2_kzz7yh-f49503.xml

```diff
@@ -205,7 +206,7 @@
                <p xml:id="kzz7yh-f49503-d1e366">
                   <lb ed="#V" n="49"/>Qui vult tenere opinionem aliam: potest 
       di<lb ed="#V" n="50" break="no"/>cere ad primum in 
-      opposi<lb ed="#V" n="51" break="no"/>tum quod quod est ipsius anime: est essentia anime concreta cum 
+      opposi<lb ed="#V" n="51" break="no"/>tum quod quod<!--maybe a <sic></sic> here?--> est ipsius anime: est essentia anime concreta cum 
```

Comment:
maybe the second "quod" is "quaedam?"

Reply:
@jeffreycwitt 
Ignore the Ade comment at the beginning of the pull request because we talked about it